app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/VideoProjectionMenuWidget.java

```diff
@@ -116,4 +117,37 @@ private void handleClick(@VideoProjectionFlags int aVideoProjection) {
     public @VideoProjectionFlags int getSelectedProjection() {
         return mSelectedProjection;
     }
+
+    public static int getAutomaticProjection(String aURL) {
+        if (aURL == null) {
+            return -1;
+        }
+
+        Uri uri = Uri.parse(aURL);
+        if (uri == null) {
+            return -1;
+        }
+
+        String projection = uri.getQueryParameter("mozVideoProjection");
```

Comment:
is there a way to allow the key name to be case insensitive? without iterating through all of the query-strings, perhaps also check `mozvideoprojection`?